elastic/kibana
# [ResponseOps] Fixing serverless tests that fail on quality gate for lastest ES Serverless image

Resolves https://github.com/elastic/kibana/issues/169204

## Summary

Fixes tests that are failing on quality gate for ES Serverless images.
Ex run where tests are failing - https://buildkite.com/elastic/kibana-elasticsearch-serverless-verify-and-promote/builds/111#018b5cd1-0ebb-4d87-9b0e-09376117a1f0


### To verify

Run and verify the tests pass
 ```
node scripts/functional_tests --bail --config x-pack/test_serverless/api_integration/test_suites/observability/common_configs/config.group1.ts --esFrom serverless --esServerlessImage latest
```
